storybook/src/pages/amsterdam/home/HomeTopTasks.tsx

```diff
@@ -0,0 +1,84 @@
+import { Card, Grid, Heading, Paragraph } from '@amsterdam/design-system-react'
+
+export const HomeTopTasks = () => (
```

Comment:
Also did not expect static content, we could randomize the content for the card story and then load it 6 times in here. The benefit of re-using stories is not being used here.

Reply:
I tried, but randomizing doesn’t work very well for pages. What we have are realistic top tasks. Randomizing them allows a task to appear multiple times, which is confusing.

I did update the Card component Stories to display a random top task.